### PR TITLE
[1LP][RFR]Replication tab conversion

### PR DESCRIPTION
**Pull request by rlbabyuk**
Purpose or Intent
=================
Convert Diagnostic Replication

Replication tab where moved to Settings->Region->Replication, also Remote type is not covered in this PR as widget should be added to cover Exclude tables area.

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit 3b2ddcc3221b2efaab85db103367e8edec4e5cb5:

`cfme/configure/configuration/diagnostics_settings.py`:
- :heavy_exclamation_mark: Line 487:1: **W391** *blank line at end of file*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

File: cfme/configure/configuration/region_settings.py [hunk 1]
```diff
@@ -870,3 +872,172 @@ class CANDUCollectionDetails(CFMENavigateStep):
...
+
+class ReplicationView(RegionView):
+    """ Replication Tab View """
+    replication_type = BootstrapSelect(id='replication_type')
+    save_button = Button('Save')
+    reset_button = Button('Reset')
+
+    @property
+    def is_displayed(self):
+        return self.replication_type.is_displayed
```
Comment (mshriver) [inline cfme/configure/configuration/region_settings.py]: **Question**

What about an additional check for the correct Region?

Comment (rlbabyuk) [reply cfme/configure/configuration/region_settings.py]: Good point, thank you. I've missed that. Will add a check for the Region

File: cfme/configure/configuration/region_settings.py [hunk 2]
```diff
@@ -870,3 +872,185 @@ class CANDUCollectionDetails(CFMENavigateStep):
...
+    """ Replication Tab View """
+    replication_type = BootstrapSelect(id='replication_type')
+    save_button = Button('Save')
+    reset_button = Button('Reset')
+
+    @property
+    def in_region(self):
+        return (
+            self.accordions.settings.tree.currently_selected == [
+                self.obj.appliance.server.zone.region.settings_string]
```
Comment (mshriver) [inline cfme/configure/configuration/region_settings.py]: :+1:

File: cfme/tests/distributed/test_appliance_replication.py [hunk 3]
```diff
@@ -74,20 +73,20 @@ def configure_db_replication(db_address, appliance):
...
+                             num_sec=360, delay=10,
                              fail_func=current_appliance.server.browser.refresh,
                              message="get_replication_status")
     assert rep_status
-    wait_for(lambda: conf.get_replication_backlog(navigate=False) == 0, fail_condition=False,
-             num_sec=120, delay=10, fail_func=current_appliance.server.browser.refresh,
-             message="get_replication_backlog")
+    wait_for(lambda: replication_conf.get_global_replication_backlog == 0, fail_condition=False,
+             num_sec=120, delay=10,
+             fail_func=current_appliance.server.browser.refresh, message="get_replication_backlog")
```
Comment (mshriver) [inline cfme/tests/distributed/test_appliance_replication.py]: I think you can drop `current_appliance` here and use `appliance` instead.

File: cfme/tests/distributed/test_appliance_replication.py [hunk 4]
```diff
@@ -6,10 +6,9 @@
 
 from cfme.base.ui import ServerView
 from cfme.common.vm import VM
-from cfme.configure import configuration as conf
 from cfme.infrastructure.provider import wait_for_a_provider
 from cfme.utils import version
-from cfme.utils.appliance import provision_appliance, current_appliance
+from cfme.utils.appliance import provision_appliance
```
Comment (mshriver) [inline cfme/tests/distributed/test_appliance_replication.py]: :tada:

**Comment by mshriver** [comment]
@izapolsk I'm tempted to merge but I'm going to wait several hours. If you get back online before dev freeze later today please review/merge this.

**Comment by mshriver** [comment]
Note PRT won't run these tests, but the moved methods are ONLY in the modified test file.  All views are new.

**Pull request merged**